ui/src/components/segments/ConstraintForm.tsx

```diff
@@ -362,7 +366,20 @@ const ConstraintForm = forwardRef((props: ConstraintFormProps, ref: any) => {
                   </label>
                 </div>
                 <div className="sm:col-span-2">
-                  <Input name="property" id="property" forwardRef={ref} />
+                  {formik.values.type === ConstraintType.ENTITY_ID ? (
```

Comment:
Quick thought on the layout here, this applies to how the UI works today too, but thought while you're here and in the light of these changes:

Should we move `type` to be before `property` here in this form? So that it is:
- type
- property
- operator
- value

Then I think instead of making this input disabled or enabled, we could just omit it when `type == entityId`. So that it just visually collapses to be:

- type
- operator
- value
 
While it kind of was acceptable before to have property then type, I think type then property makes sense in all cases?

Reply:
@GeorgeMac I am ok with this! I think to not show property with the type being entityId makes a lot of sense as well.